src/Instance.ts

```diff
@@ -15,6 +15,9 @@ import type { Token, TokensList } from './Tokens.ts';
 
 export type ResultCallback = (error: Error | null, parseResult?: string) => undefined | void;
 
+type unknownFunction = (...args: unknown[]) => unknown;
```

Comment:
```suggestion
type UnknownFunction = (...args: unknown[]) => unknown;
```

Lets use the same convention for all types (PascalCase)